fe/fe-core/src/main/java/com/starrocks/http/HttpConnectProcessor.java

```diff
@@ -94,6 +93,7 @@ protected void handleQuery() {
             // Client failed.
             LOG.warn("Process one query failed because IOException: ", e);
             ctx.getState().setError("StarRocks process failed");
```

Comment:
can we use e.getMessage() instead of "StarRocks process failed" to give a more detailed reason?

Reply:
IOException may contian sensitive message. The old logic seems doesn't want to return it to client. Maybe it's reasonable?

Reply:
> IOException may contian sensitive message. The old logic seems doesn't want to return it to client. Maybe it's reasonable?

got it. make sense